ext/enchant/enchant.c

```diff
@@ -245,17 +275,17 @@ PHP_MINFO_FUNCTION(enchant)
 /* }}} */
 
 #define PHP_ENCHANT_GET_BROKER	\
-	pbroker = (enchant_broker *)zend_fetch_resource(Z_RES_P(broker), "enchant_broker", le_enchant_broker); \
+	pbroker = Z_ENCHANT_BROKER_P(broker); \
 	if (!pbroker || !pbroker->pbroker) {	\
```

Comment:
The first check is unnecessary (also below).

Reply:
fixed